uint8 header
float32 ancor1
float32 ancor2
float32 ancor3
float32 ancor4

